industrial_reconstruction/industrial_reconstruction/industrial_reconstruction.py

```diff
@@ -278,6 +280,7 @@ def stopReconstructionCallback(self, req, res):
 
         self.get_logger().info("DONE")
         res.success = True
+        res.message = "The mesh was saved to " + str(archive_mesh_filepath)
```

Comment:
This should say the file path in `req.mesh_filepath` like is logged in line 273:
```
self.get_logger().info("Mesh Saved to " + req.mesh_filepath)
```